armonik/variables.tf

```diff
@@ -164,8 +164,50 @@ variable "admin_gui" {
   default = null
 }
 
-# Parameters of old admin gui
-variable "admin_old_gui" {
+# Deprecated, must be removed in a future version
+# Parameters of admin gui v0.8 (previously called old admin gui)
+variable "admin_0_8_gui" {
+  description = "Parameters of the old admin GUI"
+  type = object({
+    api = object({
+      name  = string
+      image = string
+      tag   = string
+      port  = number
+      limits = object({
+        cpu    = string
+        memory = string
+      })
+      requests = object({
+        cpu    = string
+        memory = string
+      })
+    })
+    old = object({
+      name  = string
+      image = string
+      tag   = string
+      port  = number
+      limits = object({
+        cpu    = string
+        memory = string
+      })
+      requests = object({
+        cpu    = string
+        memory = string
+      })
+    })
+    service_type       = string
+    replicas           = number
+    image_pull_policy  = string
+    image_pull_secrets = string
+    node_selector      = any
+  })
+  default = null
+}
+
+# Deprecated, must be removed in a future version
+variable "admin_0_9_gui" {
```

Comment:
The parameters of 0.9 should be a copy of the ones from `admin_gui`, not `admin_old_gui`.

Reply:
Yes, I change it.